docs/guides/backup_security.md

```diff
@@ -1,43 +1,46 @@
-# Backup and security
+# Backup and Security
 
 Security Checklist
```

Comment:
- This text doesn't seem to describe the TOC below.
- The TOC has no links, and could possibly be deleted entirely.

Reply:
Agreed!